2025.md

```diff
@@ -5,7 +5,192 @@ permalink: /index.html
 
 # Lucca-Medievale-2025
 
-## Lucca Medievale 2025
+## 🗣️ **Lingua**
+
+* [Italiano 🇮🇹](https://luccamedievale.it/)
+* [English 🇺🇸 🇬🇧](https://luccamedievale-it.translate.goog/?_x_tr_sl=it&_x_tr_tl=en&_x_tr_hl=it&_x_tr_pto=wapp)
```

Comment:
```suggestion
Read the [English 🇺🇸 🇬🇧](https://luccamedievale-it.translate.goog/?_x_tr_sl=it&_x_tr_tl=en&_x_tr_hl=it&_x_tr_pto=wapp) version.
```

altrimenti penso sia ok se vuoi tenere la sezione Lingua (magari rinominata in altre lingue) separata, ma comunque toglierei il link a italiano perché in pratica punta se stesso e rischi che la gente ci vada a cliccare per nulla